app/uk/gov/hmrc/plasticpackagingtax/registration/forms/liability/ExpectToExceedThresholdWeightDate.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms.liability
+
+import com.google.inject.Singleton
+import play.api.data.Form
+import uk.gov.hmrc.plasticpackagingtax.registration.config.AppConfig
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.Date
+
+import java.time.{Clock, LocalDate}
+import javax.inject.Inject
+import scala.util.Try
+
+@Singleton
+class ExpectToExceedThresholdWeightDate @Inject() (appConfig: AppConfig, clock: Clock) {
```

Comment:
We've had to use a singleton class rather than the traditional object since we need an injected AppConfig and Clock (for testing).